tests/setup.py

```diff
@@ -16,7 +16,7 @@
         'ducktape@git+https://github.com/redpanda-data/ducktape.git@cad0aeaa32da8ad43a297e50d6cdb215770fd10c',
         'prometheus-client==0.9.0', 'pyyaml==6.0', 'kafka-python==2.0.2',
         'crc32c==2.2', 'confluent-kafka==2.2.0', 'zstandard==0.15.2',
-        'xxhash==2.0.2', 'protobuf==4.21.8', 'fastavro==1.4.9',
+        'xxhash==2.0.2', 'protobuf==4.21.8', 'fastavro==1.4.9', 'falcon=3.1.3',
```

Comment:
this is not used anywhere in his patch?

Reply:
Since the verifier service will use the web server script, I've just added it here so I can run it to debug. This will be different PR at the end.